docs/user/03-traces.md

```diff
@@ -461,62 +461,70 @@ System-related spans reported by Istio are filtered out without the opt-out opti
 
 ## Troubleshooting
 
-### Traces Not Arriving at the Destination
+### No Spans Arrive at the Backend
 
-   Cause: Wrong backend endpoint configuration or authentication credentials are used.
+Cause: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
 
-   Remedy: Investigate the cause with the following steps:
-   1. Check the `telemetry-trace-collector` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+Remedy: 
+- Check the `telemetry-trace-collector` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+- Check if the backend is up and reachable.
 
-### Custom Spans don't Arrive at the Destination, but Istio Spans do
+### Not All Spans Arrive at the Backend
 
-   Cause: Your SDK version is incompatible with the OTel collector version.
+Cause: The backend is reachable and the connection is properly configured, but some spans are refused.
 
-   Remedy:
-   1. Check which SDK version you are using for instrumentation.
-   1. Investigate whether it is compatible with the OTel collector version.
-   1. If required, upgrade to a supported SDK version.
+Remedy:
+- Check the `telemetry-trace-collector` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`. If backend is refusing spans limiting the rate, try the options desribed in [Gateway Buffer Filling Up](#gateway-buffer-filling-up)
```

Comment:
This makes me suspect that "not all spans arrive" and "buffer filling up" have huge overlap and might belong to the same troubleshooting?

Reply:
As discussed, those buffer overflow can be also caused by a network latency